src/components/user-profile.component.js

```diff
@@ -32,6 +33,9 @@ export const UserProfile = ({
       />
       <Text style={styles.title}>{user.name}</Text>
       <Text style={styles.subtitle}>{initialUser.login}</Text>
+      <Text style={styles.bio}>
+        {get(initialUser, ['bio', 'length'], 0) ? initialUser.bio : user.bio}
```

Comment:
Also I think we should _merge_ `initialUser` and `user`, I think a _dumb_ user profile component should have no logic at all and should work for any user no matter if it's logged in or not. We can discuss this in another issue, I just didn't want to forget about this.